docs/reference/migration/migrate_7_0.asciidoc

```diff
@@ -31,6 +31,7 @@ Elasticsearch 6.x in order to be readable by Elasticsearch 7.x.
 * <<breaking_70_plugins_changes>>
 * <<breaking_70_analysis_changes>>
 * <<breaking_70_api_changes>>
+* <<breaking_70_java_changes>>
```

Comment:
One small thing to add: the following section needs to include the `java:asciidoc` you added, otherwise the link cannot be resolved.